src/relations.jl

```diff
@@ -221,6 +221,24 @@ vapor_specific_humidity(q::PhasePartition) = max(0, q.tot - q.liq - q.ice)
 vapor_specific_humidity(param_set::APS, ts::ThermodynamicState) =
     vapor_specific_humidity(PhasePartition(param_set, ts))
 
+"""
+    ∂q_vap_sat_∂T(param_set, ts)
+
+The change in saturation vapor specific humidity with temperature given by the Clausius Clapeyron relation
+ - `param_set` an `AbstractParameterSet`, see the [`Thermodynamics`](@ref) for more details
+ - `ts` ThermodynamicState
+"""
+function ∂q_vap_sat_∂T(param_set::APS, ts::ThermodynamicState)
+    R_v::FT = ICP.R_v(param_set)
+    L_v = latent_heat_vapor(param_set, ts)
+    L_s = latent_heat_sublim(param_set, ts)
+    λ = liquid_fraction(param_set, ts)
+    L = λ * L_v + (1 - λ) * L_s
+    T = air_temperature(param_set, ts)
+    q_vap_sat = vapor_specific_humidity(param_set, ts)
+    return L * q_vap_sat / (R_v * T^2)
```

Comment:
```suggestion
    return q_vap_sat * (L / (R_v * T^2) - 1/T)
```
(See updated design doc. The second term was missing. We should use this function, once updated, in saturation adjustment iterations too.)